ludwig/profiling/proto/gen-proto.sh

```diff
@@ -0,0 +1,6 @@
+protoc --go_out=. --python_out=. \
```

Comment:
May want to consider using [betterproto](https://github.com/danielgtaylor/python-betterproto) to produce cleaner dataclasses as output instead of the default python_out - eg:
```
protoc --python_betterproto_out=. -I ./ludwig/profiling/proto/ whylogs_messages.proto dataset_profile.proto
```

Reply:
I really like betterproto, but unfortunately it doesn't support [`Any` message types](https://developers.google.com/protocol-buffers/docs/proto3#any).